src/polymer/expression-scanner.ts

```diff
@@ -59,6 +59,11 @@ export function getAllDataBindingTemplates(node: parse5.ASTNode) {
 
 export type HtmlDatabindingExpression =
     TextNodeDatabindingExpression | AttributeDatabindingExpression;
+
+// Some expressions are limited. For example, in a property declaration,
```

Comment:
Looks like they are, but I didn't have a good reason for doing so.

Switched to `/** docs */` style